packages/ui-components/src/Charts/barChart.module.scss

```diff
@@ -0,0 +1,38 @@
+@import "Style/util/color.scss";
+@import "Style/util/spacing.scss";
+@import "Style/util/styling.scss";
+@import "Style/util/fonts.scss";
+@import "Style/util/type.scss";
+@import "Style/util/depth.scss";
```

Comment:
So, none of this is going to work in this repo. I don't believe you are using all of these imports anyway (is that common in managed-service to over import the utilities?), but for the mixins you are using, you could port them over or use the existing design tokens and put the styles in place.

Reply:
Ah whoops! Totally blanked while adding these. I removed unnecessary imports and was able to get away without one of the types mixins.